From fb5168bc0876c5acc74d18c89a9c5a78578112e5 Mon Sep 17 00:00:00 2001
From: Pedro Palazon Candel <pedro@joyent.com>
Date: Thu, 23 Nov 2017 18:21:43 +0100
Subject: [PATCH] HEAD-2377 SAPI zone should use its own admin IP address for
 sapi-url, not sapi's DNS domain Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
 Approved by: Josh Wilsdon <josh@wilsdon.ca>

---
 scripts/headnode.sh | 1 -
 1 file changed, 1 deletion(-)

diff --git a/scripts/headnode.sh b/scripts/headnode.sh
index 7e601941..ce58f72b 100755
--- a/scripts/headnode.sh
+++ b/scripts/headnode.sh
@@ -638,7 +638,6 @@ upload_values
 download_metadata
 write_initial_config
 registrar_setup
-mdata-put sapi-url http://${CONFIG_sapi_domain}
 HERE
         setup_state_add "sapi_bootstrapped"
     fi
-- 
2.21.0

